core/src/main/java/org/apache/carbondata/core/dictionary/generator/key/DictionaryKey.java

```diff
@@ -16,12 +16,12 @@
  */
 package org.apache.carbondata.core.dictionary.generator.key;
 
-import java.io.Serializable;
+import io.netty.buffer.ByteBuf;
 
 /**
  * Dictionary key to generate dictionary
  */
-public class DictionaryKey implements Serializable {
+public class DictionaryKey {
```

Comment:
Ok, but added in same object to avoid other unnecessary copying of metadata from request to response